### PR TITLE
doc, Makefile: fix and update the operatorhub instruction related matters 

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
commit 1:
Makefile: add package manifests target directory 

commit 2:
doc: fix operatorhub publishing instruction

There was a mistake in #842 that 'git clone' command didn't come up
together with the command 'cp -r'. In addition, this pr includes
the changes following [the suggestion made after #842 was merged](https://github.com/intel/intel-device-plugins-for-kubernetes/pull/842#discussion_r782058383).

Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

### CODE REVIEW

File: DEVEL.md [hunk 1]
```diff
@@ -202,14 +202,14 @@ Update metadata.annotations.containerImage and metadata.annotations.createdAT fi
...
 ```
-$ make packagemanifests OPERATOR_VERSION=0.X.Y
+$ git clone https://github.com/<GitHub Username>/community-operators
 ```
 
-Copy the generated files to the Community Operators repo:
+Generate package manifests with:
 ```
-$ cp -r packagemanifests/* community-operators/operators/intel-device-plugins-operator/
+$ make packagemanifests OPERATOR_VERSION=0.X.Y PACKAGEMANIFESTS_DIR=/community-operators/operators/intel-device-plugins-operator/
```
Comment (bart0sh) [inline DEVEL.md:212]:
```suggestion
$ make packagemanifests OPERATOR_VERSION=0.X.Y PACKAGEMANIFESTS_DIR=community-operators/operators/intel-device-plugins-operator/
```

Comment (bart0sh) [reply DEVEL.md:212]: PACKAGEMANIFESTS_DIR=community-operators/operators/intel-device-plugins-operator/ is a long line. Can we drop it?

**Comment by bart0sh** [comment]
@hj-johannes-lee please fix typo in the commit subject:
doc: fix opeatorhub publishing instruction - > doc: fix operatorhub publishing instruction

File: DEVEL.md (hunk 1 above)
Comment (hj-johannes-lee) [reply DEVEL.md]: @bart0sh Ok, I just dropped the whole part.!

**Comment by hj-johannes-lee** [comment]
@bart0sh Ok, I fixed the typo.!

**Review by bart0sh — APPROVED** [review]
/lgtm

File: Makefile [hunk 2]
```diff
@@ -119,10 +120,10 @@ bundle:
 .PHONY: packagemanifests
 packagemanifests:
 	$(OPERATOR_SDK) generate kustomize manifests -q --input-dir $(OLM_MANIFESTS) --output-dir $(OLM_MANIFESTS) --apis-dir pkg/apis
-	$(KUSTOMIZE) build $(OLM_MANIFESTS) | sed "s|intel-deviceplugin-operator:devel|intel-deviceplugin-operator:$(OPERATOR_VERSION)|" | $(OPERATOR_SDK) generate packagemanifests -q --kustomize-dir $(OLM_MANIFESTS) --version $(OPERATOR_VERSION) --from-version $(OPERATOR_PREVIOUS_VERSION) $(BUNDLE_METADATA_OPTS)
-	# Remove unneeded resources
-	rm packagemanifests/$(OPERATOR_VERSION)/*service.yaml
-	rm packagemanifests/$(OPERATOR_VERSION)/*clusterrole.yaml
+	$(KUSTOMIZE) build $(OLM_MANIFESTS) | sed "s|intel-deviceplugin-operator:devel|intel-deviceplugin-operator:$(OPERATOR_VERSION)|" | $(OPERATOR_SDK) generate packagemanifests -q --kustomize-dir $(OLM_MANIFESTS) --version $(OPERATOR_VERSION) --from-version $(OPERATOR_PREVIOUS_VERSION) $(BUNDLE_METADATA_OPTS) --output-dir $(PACKAGEMANIFESTS_DIR)
+	# Remove unneeded resourcess
```
Comment (mythi) [inline Makefile:124]:
```suggestion
	# Remove unneeded resources
```

Comment (hj-johannes-lee) [reply Makefile]: Idk why I always make such "unneeded" typos.. Sorry, let me be more careful!

**Review by bart0sh — APPROVED** [review]
/lgtm

**Review by mythi — APPROVED** [review]

**Pull request merged**